server/game/cards/13.5-TB/Yoren.js

```diff
@@ -0,0 +1,28 @@
+const DrawCard = require('../../drawcard.js');
+
+class Yoren extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardEntersPlay: event => event.card === this
+            },
+            target: {
+                cardCondition: card => card.getType() === 'character' && card.owner !== this.controller && card.getPrintedCost() <= 5
```

Comment:
You'll need to check that the location is 'play area', otherwise it'll allow players to take control of cards in the opponent discard / dead piles.